playbooks/roles/prometheus/meta/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+dependencies:
+  - name: ernestas-poskus.prometheus
+    prometheus_node_exporter_install: false
+  - name: nginx-proxy
+    NGINX_PROXY_CONFIG_PATH: /etc/nginx/sites-available/prometheus
+    NGINX_PROXY_PUBLIC_PORT: 443
+    NGINX_PROXY_INTERNAL_PORT: 9090
+    NGINX_PROXY_USERNAME: "{{ PROMETHEUS_BASIC_AUTH_USERNAME }}"
+    NGINX_PROXY_PASSWORD: "{{ PROMETHEUS_BASIC_AUTH_PASSWORD }}"
+  - name: nginx-proxy
+    NGINX_PROXY_CONFIG_PATH: /etc/nginx/sites-available/grafana
```

Comment:
I'm not sure how much the Grafana task will expand, but if it's going to expand quite a bit if we install stuff through Ansible for it, maybe it should be its own role?

Reply:
I'm not sure yet whether we will be able to completely configure it with Ansible.  If this turns out to be easy to do, I will do it, but otherwise I will simply manually configure it and configure a backup for the configuration, so we can restore it if we lose the server.

Reply:
Sounds good :+1: